examples/reactnative/rn063example/android/app/src/main/java/com/rn063example/MainActivity.java

```diff
@@ -1,5 +1,7 @@
 package com.rn063example;
 
+import android.os.Bundle;
```

Comment:
Actually, this seems to have been unnecessary - will remove.

Reply:
...although having said that I've then added code to `onCreate` in the next PR to come, so I may as well leave it in.